src/rust/ensogl/src/display/shape/primitive/shader/canvas.rs

```diff
@@ -280,6 +280,30 @@ impl Canvas {
             shape
         })
     }
+
+    /// Grow the shape by the given value.
+    pub fn grow<T:Into<Var<f32>>>
+    (&mut self, num:usize, s:Shape, value:T) -> Shape {
+        self.if_not_defined(num, |this| {
+            let value:Glsl = value.into().glsl();
+            let expr = iformat!("return grow({s.getter()},{value});");
+            let mut shape = this.new_shape_from_expr(num,&expr);
+            shape.add_ids(&s.ids);
+            shape
```

Comment:
I see that my code is broken as well (maybe because of some refactoring), but please, keep the proper alignment in new code :)